bugsnag-android-core/src/test/java/com/bugsnag/android/DocumentPathTest.kt

```diff
@@ -0,0 +1,203 @@
+package com.bugsnag.android
+
+import com.bugsnag.android.internal.DocumentPath
+import org.junit.Assert
+import org.junit.Test
+import java.util.LinkedList
+import kotlin.collections.HashMap
+
+class DocumentPathTest {
```

Comment:
The test still explicitly specifies type in a lot of places, e.g.:

`mutableMapOf<String, Any>("a" to LinkedList<Any>(listOf<Any>(1, 2, 3)))`

Readability would be improved by getting rid of this where possible throughout this file. For example, that specific test case could be written like this instead:

`mutableMapOf("a" to LinkedList(listOf(1, 2, 3)))`

Reply:
OK I think I got them all.